airbyte-cdk/java/airbyte-cdk/core/build.gradle

```diff
@@ -63,8 +69,7 @@ dependencies {
     implementation libs.hikaricp
     implementation libs.bundles.debezium.bundle
 
-    implementation libs.bundles.datadog
-    // implementation 'com.datadoghq:dd-trace-api'
+    api libs.bundles.datadog
```

Comment:
Thanks. In case you haven't noticed, the dependencies need to be cleaned up a bit. I don't see us doing this any other way than little piece by little piece. Every little bit helps.